velox/exec/Driver.cpp

```diff
@@ -479,10 +479,18 @@ StopReason Driver::runInternal(
               }
               RuntimeStatWriterScopeGuard statsWriterGuard(op);
               if (op->isFinished()) {
+                // This branch is to update the finishTiming of op.
                 auto timer =
```

Comment:
Here, we create timer, but do not do any work. There is no point in timing nothing. Please, revise.

Reply:
Yes. It seems doesn't look right. I don't know how to record the current op's `finishTiming`. Do you have any suggestion? Thanks. @mbasmanova @rui-mo

Reply:
@JkSelf finishTiming measures the time spend in Operator::noMoreInput. The name is confusing. It comes from Presto where Operator has finish() method instead of noMoreInput.

Reply:
@mbasmanova  It seems there is no `finish()` method in Operator. And I still don't understand how to calculate this time.

Reply:
Yes, there is no finish() method. finishTiming (naming is bad) measures time spend in noMoreInput().